stories/index.js

```diff
@@ -205,3 +205,11 @@ storiesOf('ChipInput', module)
       addOnBlur
     />
   ))
+  .add('single form field', () => themed(
+    <form>
```

Comment:
Allright. :+1: Could you please add a text to this story? Something like "This is a chip input inside a `<form>`. Notice how the form is _not_ submitted when you press _Enter_ inside the chip input."